orion/visualization.py

```
@@ -26,6 +26,7 @@ def __init__(
         change_points_by_metric: dict,
         uuid_field: str,
         version_field: str,
+        algorithm_name: str = "",
         acked_entries: list = None,
```

Ruff (0.16.2)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)